packages/swapper/src/swappers/ThorchainSwapper/utils/getBestAggregator.ts

```diff
@@ -65,7 +65,7 @@ export const getBestAggregator = async (
   const bestContractData = bestPool ? poolContractData.get(bestPool) : undefined
   const bestAggregator = bestContractData ? feeAmountToContractMap[bestContractData.fee] : undefined
 
-  if (!bestAggregator || !quotedAmountOut) {
+  if (!bestAggregator || quotedAmountOut === undefined) {
```

Comment:
Gucci in theory but... Do we want to allow 0 as a quoted amount out? Probably not. Let's do explicit undefined *and* 0 checks.